README.md

```diff
@@ -38,7 +50,13 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```The rules of scope establish the difference between local and global variables;
+i.e., when a given value or definition for a variable applies.
+Any variable definition within a function is local, and applies only within that function.
+Outside of a function, variables created without the var prefix (example: "bread = 'wheat'")
```

Comment:
Any variable anywhere -- inside or outside a function -- that doesn't have `var` becomes a global variable.